Trembeczki exemption list (#197)
mulesoft/docs-access-management: modules/ROOT/pages/multi-factor-authentication.adoc

Service users or accounts that make programmatic calls to Anypoint Platform should be exempt from multi-factor authentication because they cannot provide verification aside from a username and password.

[NOTE]

> The Exempt Accounts list appears only if multi-factor authentication is required in your entire organization.

Reviewer: ```suggestion
The Exempt Accounts list appears only if your organization requires multi-factor authentication.
```

> The Exempt Accounts list only appears if the Multi-Factor authentication option is set to Required inside the Identity Provider's settings

Reviewer: ```suggestion
The Exempt Accounts list appears only if multi-factor authentication is required in your entire organization.
```
